dm/pkg/terror/error_list.go

```diff
@@ -195,6 +195,9 @@ const (
 	// pkg/shardddl/optimism.
 	codeShardDDLOptimismNeedSkipAndRedirect
 	codeShardDDLOptimismAddNotFullyDroppedColumn
+
+	// syncer.
+	codeCancelledDDL
```

Comment:
Will move to sync unit error code(36000+) be better?

Reply:
I have renamed the error to `codeSyncerCancelledDDL` and moved it to sync unit error code.